src/vs/workbench/contrib/debug/browser/debugAdapterManager.ts

```diff
@@ -202,12 +202,15 @@ export class AdapterManager extends Disposable implements IAdapterManager {
 						type: 'boolean',
 						description: nls.localize('suppressMultipleSessionWarning', "Disable the warning when trying to start the same debug configuration more than once."),
 						default: true
+					},
+					'extends': {
+						type: 'string'
 					}
 				}
 			}
 		};
 		launchSchema.definitions = definitions;
-		items.oneOf = [];
+		items.oneOf = [{ type: 'object', properties: { extends: { type: 'string' } }, required: ['extends'] }];
```

Comment:
I don't quite understand oneOf or why it was empty before. But now this is saying that an object with `extends` is valid right?